### PR TITLE
Rename checkout target customization to order submission

### PR DESCRIPTION
**Pull request by rrxxww**

### CODE REVIEW

**Comment by nicolae-stroncea** [comment]
`Shopify/function-examples/.graphqlrc.cjs` has this:


```
const projects = {
    ...getProjects("sample-apps/discounts/extensions"),
    ...getProjects("sample-apps/payment-customizations/extensions"),
    ...getProjects("sample-apps/delivery-customizations/extensions"),
    ...getProjects("checkout/rust/completion-target-customization"),
    ...getProjects("checkout/rust/delivery-customization"),
    ...getProjects("checkout/rust/payment-customization"),
    ...getProjects("checkout/javascript/completion-target-customization"),
    ...getProjects("checkout/javascript/delivery-customization"),
    ...getProjects("checkout/javascript/payment-customization"),
    ...getProjects("checkout/javascript/cart-checkout-validation"),
    ...getProjects("discounts/rust/order-discounts"),
    ...getProjects("discounts/rust/product-discounts"),
    ...getProjects("discounts/rust/shipping-discounts"),
}
```

Should this be updated or does it automatically get updated?

File: yarn.lock [hunk 1]
```diff
@@ -47,6 +47,14 @@
   dependencies:
     "@babel/highlight" "^7.18.6"
 
+"@babel/code-frame@^7.18.6", "@babel/code-frame@^7.22.13":
```
Comment (drobin03) [inline yarn.lock:50]: @rrxxww was this change intentional?

**Review by drobin03 — APPROVED** [review]
Changes LGTM - just a suspicious yarn.lock change that I think was unintended.

**Comment by rrxxww** [comment]
> Changes LGTM - just a suspicious yarn.lock change that I think was unintended.

Nice catch, thanks!
